packages/plugin-team/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@ciscospark/plugin-team",
+  "version": "0.7.0-alpha",
+  "description": "",
+  "license": "MIT",
+  "author": "Josh Dykstra <jodykstr@cisco.com>",
+  "main": "dist/index.js",
+  "devMain": "src/index.js",
+  "repository": "https://github.com/ciscospark/spark-js-sdk/tree/master/packages/plugin-team",
+  "dependencies": {
+    "@ciscospark/common": "^0.7.0-alpha",
+    "@ciscospark/plugin-conversation": "^0.7.0-alpha",
```

Comment:
The teams plugin uses a lot of the conversation code, actually. Things like adding a user to a team is really just like adding a user to a conversation, so we just call the conversation method under the hood. Because we're relying on the conversation, encryption, and user plugins in the source and not just the test files these are dependencies.